### PR TITLE
[#160556015] Rotation for cc db encryption key - Part 2

### PR DESCRIPTION
**Pull request by henrytk**
What
----

This removes the temporary rotation pipeline added in #1597 

How to review
-------------

Code review should be sufficient.

Who can review
--------------

Anyone but me.

### CODE REVIEW

File: concourse/pipelines/create-cloudfoundry.yml [hunk 1]
```diff
@@ -3250,6 +3251,37 @@ jobs:
         params:
           file: updated-tfstate/cf.tfstate
 
+  - name: rotate-cc-db-encryption-key
+    serial: true
+    plan:
+    - aggregate:
+      - get: paas-cf
```
Comment (alext) [inline concourse/pipelines/create-cloudfoundry.yml]: We should probably pin this to the version thats passed cf-deploy to ensure we're using a version that matches what's been deployed...

Comment (henrytk) [reply concourse/pipelines/create-cloudfoundry.yml]: Sensible, but paas-cf is only imported for the use of the `./paas-cf/concourse/scripts/bosh_login.sh` script, which is stable; additionally, this pipeline is temporary and will be removed immediately after it is run everywhere. I gave some thought to whether it is worth keeping, but it means speculating about our future implementation of the rotation mechanism.

Comment (henrytk) [reply concourse/pipelines/create-cloudfoundry.yml]: Also, the diff here is weird at the moment because it is a branch off of a branch.

**Comment by alext** [comment]
I'm going to close this for now as we're not going to run this task in the pipeline until after the next CF ugprade.

**Pull request merged**